src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -73,7 +73,7 @@ public function createListFromChoices($choices, $value = null)
             @trigger_error('Passing callable strings is deprecated since Symfony 3.1 and PropertyAccessDecorator will treat them as property paths in 4.0. You should use a "\Closure" instead.', E_USER_DEPRECATED);
         }
 
-        if ($value instanceof PropertyPath) {
+        if ($value instanceof PropertyPathInterface) {
```

Comment:
This would count as a new feature to me.

Reply:
Agree. I just thought it happened by mistake.

Reply:
reverted, will do a separate PR